.golangci.yml

```diff
@@ -130,8 +130,8 @@ linters:
   disable-all: true
   enable:
 #    - cyclop
-    - deadcode
-    - depguard
+#    - deadcode
+#    - depguard
```

Comment:
Will you return them later? :)

Reply:
deadcode is deprecated since 1.49 (now 1.56)
depguard - disabled in ydb-go-sdk (no work normaly)